pipe_tools/airflow/__init__.py

```diff
@@ -0,0 +1,3 @@
+# Monkey patch TaskInstance to get tests to run -- probably a better solution
+from airflow.models import TaskInstance
+TaskInstance.max_tries = 16
```

Comment:
@bitsofbits a bit more of detail on the comment of what the lucky `16` number does would be great.